doc/news/_preparation_next_release.md

```diff
@@ -193,75 +170,54 @@ The following text lists news about the [plugins](https://www.libelektra.org/plu
 
 ### logchange
 
-- Made logchange a notification-send hook plugin _(Maximilian Irlinger @atmaxinger)_
-
-### website
-
-- Fix broken `/pythongen` link on homepage _(@stefnotch)_
-- Fix redirect logic to not cause loops _(@stefnotch)_
-- Remove duplicated link to `TESTING.md` file _(@stefnotch)_, _(@janldeboer)_
-- Restructure parts of the links on the website _(@stefnotch)_, _(@janldeboer)_
-- Removed broken links to packages for Linux distributions. _(@Dynamichost96)_
```

Comment:
Why are all the lines above removed? Are they leftovers from a previous version?

Reply:
They were moved from the plugin section to the website section below ...